src/plugins/mer/mergeneraloptionswidget.cpp

```diff
@@ -91,5 +199,42 @@ QString MerGeneralOptionsWidget::searchKeywords() const
     return keywords;
 }
 
+void MerGeneralOptionsWidget::on_addProfileButton_clicked()
+{
+    bool ok;
+    const QString name = QInputDialog::getText(this, tr("Create profile"),
```

Comment:
We use the term "Device model" currently.

Reply:
Renamed.